Cookbook/Technical-Documents/FeatureSwitches.md

```diff
@@ -0,0 +1,167 @@
+# Feature Switches
+
+This document describes the process that should be followed when defining Features Switches.
+
+Depending on the use case we are using different ways to define Feature Switches in our code base. A feature switch is not always some `Bool` value that can be either `true` or `false` (though most of the times it will be), it can be a more complicated type that defines a conditional behaviour.
+
+1. **Local feature switches**
+
+	These feature switches are defined in the `LocalFeatureSwitchesProtocol` and are stored in `UserDefaults`. **These feature switches are supposed to be used only during the development of any new feature so that the code for the feature can be integrated into the `develop` branch continuously.** As they are stored in `UserDefaults` they can be overridden during testing with Settings app. To add a new local feature switch do the following:
+
+	1. Add a new case in the `SettingsKeys` enum for this feature flag in `LocalFeatureSwitches.swift`:
```

Comment:
Is it worth mentioning the path of the file since there are multiple named like this? Or rename the others to mention the context (e.g. `LocalFeatureSwitches+BUPA.swift`